packages/editor/src/client/panel/teaserEditPanel.tsx

```diff
@@ -36,7 +37,7 @@ export function TeaserEditPanel({
   onClose,
   onConfirm,
   closeLabel = 'Close'
-}: TeaserEditPanelProps) {
+}: TeaserEditPanelProps): JSX.Element {
```

Comment:
Why did you add this?

Reply:
@roosnic1 FYI our linting sometimes (maybe it is just my vscode) requires return type and highlights as warning. In another project we add this return type to every FC to satisfy this constraint. @bishoymelek I think we do not require return type for FC here. But maybe we have to either fix lint rules somewhere or strictly request to have consistency within the repo and the linting.

Reply:
@faebebin and @bishoymelek you might have to configure your editors to read the eslint config in the `package.json`

Reply:
@roosnic1 yes if you never saw this then will check my config.

Reply:
@roosnic1 @roosnic1 I checked my local settings. I have eslint activated, but that is project setting,no? Just, I did not find some setting to change by now. Don't you have those highlightings requiring return type:
![image](https://user-images.githubusercontent.com/21113500/105489405-76fb1580-5cb3-11eb-9a5d-38b1731a6974.png)

Reply:
And when runnging: yarn lint:editor

![image](https://user-images.githubusercontent.com/21113500/105489571-b9245700-5cb3-11eb-89a6-1868d7c95079.png)

Reply:
? If this is on me I will keep on working on my local settings :)

Reply:
@roosnic1 @faebebin Yes I guess it's the project's eslint configuration that always makes these warnings, I actually thought to run linting before git commit so it's always aligned correctly.